src/utils/idpLoader.ts

```diff
@@ -0,0 +1,55 @@
+import { errorsToReadableMessages } from "italia-ts-commons/lib/reporters";
+import nodeFetch from "node-fetch";
+import { DOMParser } from "xmldom";
+import { IDPEntityDescriptor } from "../types/IDPEntityDescriptor";
+import { log } from "./logger";
+
+const EntityDescriptorTAG = "md:EntityDescriptor";
+const X509CertificateTAG = "ds:X509Certificate";
+const SingleSignOnServiceTAG = "md:SingleSignOnService";
+const SingleLogoutServiceTAG = "md:SingleLogoutService";
+
+export async function parseIdpMetadata(
```

Comment:
Please add a comment that explain the purpose of this

Reply:
fixed in 6581c21

Reply:
that commit adds a comment to `mapIpdMetadata`

Reply:
Right. I've add comment for `parseIdpMetadata` and `fetchIdpMetadata` in commit 26b860c